src/prefect/events/clients.py

```diff
@@ -148,6 +151,12 @@ async def emit(self, event: Event) -> None:
         finally:
             EVENTS_EMITTED.labels(self.client_name).inc()
 
+    def sync_raise_for_connection_error(self):
```

Comment:
for methods like this we are trying to adopt a naming convention like:
```
def raise_for_connection_error(self):
...

async def araise_for_connection_error(self):
...
```

Reply:
Ha, jinx!